src/instance.ts

```diff
@@ -388,19 +463,25 @@ class Instance extends common.ServiceObject {
    *   const exists = data[0];
    * });
    */
-  exists(callback) {
+  exists(
+    callback?: ExistsInstanceCallback
+  ): void | Promise<ExistsInstanceResponse> {
     const NOT_FOUND = 5;
 
     this.getMetadata(err => {
       if (err && err.code !== NOT_FOUND) {
-        callback(err, null);
+        callback!(err, null);
         return;
       }
 
       const exists = !err || err.code !== NOT_FOUND;
-      callback(null, exists);
+      callback!(null, exists);
     });
   }
+
+  get(options?: GetConfig): Promise<GetInstanceResponse>;
```

Comment:
I might advocate that we should extend `GetConfig` and create an interface called `GetInstanceConfig`. In the event that Spanner decides to add options later, this would make it a little more future proof and prevent us from having to make a breaking change. WDYT?

Reply:
sounds good...